src/services/services.ts

```diff
@@ -1536,26 +1573,166 @@ namespace ts {
             return checker.getSymbolAtLocation(node);
         }
 
+
+        const sourceMapCommentRE = /^\/\/[@#] sourceMappingURL=(.+)$/gm;
+        const dataURLRE = /^data:/;
+        const base64URLRE = /^data:application\/json;charset=utf-8;base64,(.+)$/;

```

Comment:
"URLRE" is a lot of sequential uppercase letters. I'd prefer a name like `base64UrlRegExp`.